lighthouse-cli/bin.ts

```diff
@@ -175,10 +176,22 @@ export async function runLighthouse(
     url: string, flags: Flags, config: Object|null): Promise<{}|void> {
   let launchedChrome: LaunchedChrome|undefined;
 
+  if (typeof flags.disableErrorReporting === 'undefined') {
```

Comment:
that makes opt-out (which is the only usable flag) become `--no-enable-error-reporting` which feels odd, admittedly it definitely feels odd to check `disableErrorReporting` in runner but I didn't think we had any other case where we renamed the CLI flag from CLI to core